src/components/Filter.jsx

```diff
@@ -0,0 +1,171 @@
+import React, { useState, useEffect } from 'react'
+import axios from 'axios'
+import { FaFilter } from 'react-icons/fa'
+
+import '../styles/Filter.css'
+
+const Filter = ({mnag, setManga}) => {
+  const [isChecked, setIsChecked] = useState({
+    lowtohigh: false,
+    hightolow: false,
```

Comment:
Just as suggestion: Be consistent with variable naming, if you are using camelCase then stick with it.